Patches/Revia Race/ThingDefs_Misc/Weapons_Guns.xml

```diff
@@ -532,6 +532,66 @@
                   <MeleeDodgeChance>0.67</MeleeDodgeChance>
                 </equippedStatOffsets>
               </value>
+            </li>
+            			<!-- ==========  Shatterer  =========== -->
+            <li Class="PatchOperationReplace">
+              <xpath>Defs/ThingDef[defName="ReviaWarPick"]/tools</xpath>
+              <value>
+                <tools>
+                  <li Class="CombatExtended.ToolCE">
+                    <label>handle</label>
+                    <capacities>
+                      <li>Blunt</li>
+                    </capacities>
+                    <power>7</power>
+                    <cooldownTime>2.46</cooldownTime>
+                    <armorPenetrationBlunt>1.75</armorPenetrationBlunt>
+                    <chanceFactor>0.15</chanceFactor>
+                    <linkedBodyPartsGroup>Handle</linkedBodyPartsGroup>
+                  </li>
+				  <li Class="CombatExtended.ToolCE">
+					<label>head</label>
+					<capacities>
+						<li>Demolish</li>
+					</capacities>
+					<power>22</power>
+					<cooldownTime>2.56</cooldownTime>
+					<chanceFactor>0.55</chanceFactor>
```

Comment:
Would it make sense to reduce the chance factor of this to a bit less than the 'Gash' attack, since this attack seems to be mainly intended for destroying structures?

Reply:
Done